packages/quantic/force-app/main/default/lwc/quanticBreadcrumb/quanticBreadcrumb.js

```diff
@@ -0,0 +1,177 @@
+    this.dateFacetBreadcrumbs = this.breadcrumbManager.state.dateFacetBreadcrumbs;
+    this.hasBreadcrumbs = this.breadcrumbManager.state.hasBreadcrumbs;
+  }
+
+  @api
+  deselectAll() {
+    this.breadcrumbManager.deselectAll();
+    this.expandedBreadcrumbFieldsState = [];
+  }
+
+  /** @param {import("coveo").BreadcrumbValue | import("coveo").CategoryFacetBreadcrumb} breadcrumb */
+  @api
+  deselectBreadcrumb(breadcrumb) {
+    this.breadcrumbManager.deselectBreadcrumb(breadcrumb);
+  }
+
+  formatRangeBreadcrumbValue(breadcrumb) {
+    return {
+      ...breadcrumb,
+      values: (breadcrumb.values.map(range => ({
+        ...range,
+        value: `${range.value.start} - ${range.value.end}`
+      })))
+    };
+  }
+
+  formatCategoryBreadcrumbValue(breadcrumb) {
+    console.log(JSON.stringify(breadcrumb));
+    if (breadcrumb.path.length <= 3) {
+      return breadcrumb.path.map((breadcrumbValue) => breadcrumbValue.value);
+    }
+    const collapsed = '...';
+    const firstBreadcrumbValue = breadcrumb.path[0].value;
+    const lastTwoBreadcrumbsValues = breadcrumb.path
+      .slice(-2)
+      .map((breadcrumbValue) => breadcrumbValue.value);
+    return [firstBreadcrumbValue, collapsed, ...lastTwoBreadcrumbsValues];
+  }
+
+  formatDate(dateValue) {
+    const date = new Date(dateValue);
+    return date.toLocaleDateString();
+  }
+
+  formatDateRangeBreadcrumbValue(breadcrumb) {
+    return {
+      ...breadcrumb,
+      values: breadcrumb.values.map(range => ({
+        ...range,
+        value: `${this.formatDate(range.value.start)} - ${this.formatDate(range.value.end)}`
+      }))
+    };
+  }
+
+  resetExpandedBreadcrumbFieldState(breadcrumb) {
+    if (breadcrumb.values.length <= this.collapseThreshold) {
+      const newState = [...this.expandedBreadcrumbFieldsState];
+      newState.splice(
+        this.expandedBreadcrumbFieldsState.indexOf(breadcrumb.field),
+        1
+      );
+      this.expandedBreadcrumbFieldsState = newState;
+    }
+  }
+
+  getBreadcrumbValuesCollapsed(breadcrumb) {
+    return ({
+      ...breadcrumb,
+      values: breadcrumb.values.slice(0, this.collapseThreshold),
+      showMoreButton: true,
+      showMoreButtonText: `${breadcrumb.values.length - this.collapseThreshold} more...`,
+      expandButtonClick: () => {
+        this.expandedBreadcrumbFieldsState = [...this.expandedBreadcrumbFieldsState, breadcrumb.field];
+      }
+    });
+  }
+
+  getShouldCollapseBreadcrumbValues(breadcrumb) {
+    this.resetExpandedBreadcrumbFieldState(breadcrumb);
+    return breadcrumb.values.length > this.collapseThreshold && !this.expandedBreadcrumbFieldsState.includes(breadcrumb.field);
+  }
+
+  getBreadcrumbValues(breadcrumb) {
+    return (this.getShouldCollapseBreadcrumbValues(breadcrumb) ?
+      this.getBreadcrumbValuesCollapsed(breadcrumb) :
+      breadcrumb);
+  }
+
+  get facetBreadcrumbValues() {
+    let facetBreadcrumbsToDisplay = this.facetBreadcrumbs || [];
```

Comment:
```suggestion
    const facetBreadcrumbsToDisplay = this.facetBreadcrumbs || [];
```

Reply:
Yea good catch, I originally had something that was changing the value of these 👍